public/locales/es/addScholarship.json

```
@@ -0,0 +1,6 @@
+{
+  "titleTag": "Agrege una beca",
```

Website actions in Spanish generally use the infinitive:
```suggestion
  "titleTag": "Agregar una beca",
```